app/uk/gov/hmrc/eoricommoncomponent/frontend/views/start_subscribe.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.html._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.models.Service
+@import uk.gov.hmrc.eoricommoncomponent.frontend.views.ServiceName
+@import uk.gov.hmrc.eoricommoncomponent.frontend.controllers.routes.ApplicationController
+@import uk.gov.hmrc.eoricommoncomponent.frontend.config.AppConfig
+
+
+@this(layout_di: layout, appConfig: AppConfig)
+@(service: Service)(implicit messages: Messages, request: Request[_])
+
+@link(text: String, url: String) = {
+    <a href="@url" target="_blank">@messages(text)</a>
+}
+
+@layout_di(messages("cds.subscription.information.title")) {
+
+    <div class="column-two-thirds start_page">
+        <h1 class="heading-xlarge">@messages("cds.subscription.information.heading", service.friendlyName)</h1>
```

Comment:
@PatrykRudnicki Noticed this bug, so before I fix it....

Do not use `service. friendlyName` directly from the views - it won't be translated.

Use `ServiceName.longName(service)` or just `ServiceName.longName` if the service can be extracted from the implicit request.